test/new-e2e/pkg/environments/host.go

```diff
@@ -16,10 +16,11 @@ import (
 type Host struct {
 	AwsEnvironment *aws.Environment
 	// Components
-	RemoteHost *components.RemoteHost
-	FakeIntake *components.FakeIntake
-	Agent      *components.RemoteHostAgent
-	Updater    *components.RemoteHostUpdater
+	RemoteHost      *components.RemoteHost
+	FakeIntake      *components.FakeIntake
+	Agent           *components.RemoteHostAgent
+	Updater         *components.RemoteHostUpdater
+	ActiveDirectory *components.RemoteActiveDirectory
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I feel like we are adding too many things to the standard environment

Reply:
Since you don't need it in exports, as you don't have a client interacting with it, you can skip exporting it here